.github/workflows/python-app.yml

```diff
@@ -26,14 +26,14 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8
+        pip install flake8 black
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
     - name: Lint with flake8
       run: |
-        # stop the build if there are Python syntax errors or undefined names
-        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+        flake8
+    - name: Check with black
+      run: |
+        black --check --line-length 79 --verbose .
```

Comment:
Why verbose? In case of failure Black will show you the unformatted file

Reply:
Black will show an unformatted file name with or without --verbose. But I agree, we need only exit code.

Reply:
> with or without --verbose
Yes, that's what I mean :)